libraries/SoftwareSerial/examples/SoftwareSerialExample/SoftwareSerialExample.ino

```diff
@@ -0,0 +1,55 @@
+/*
+  Software serial multple serial test
+
+ Receives from the hardware serial, sends to software serial.
+ Receives from software serial, sends to hardware serial.
+
+ The circuit:
+ * RX is digital pin 10 (connect to TX of other device)
+ * TX is digital pin 11 (connect to RX of other device)
+
+ Note:
```

Comment:
I would suggest removing the comments about the allowed pins on the AVR boards on lines 11-19, since it's not relevant here.